src/components/helper/index.ts

```diff
@@ -0,0 +1,7 @@
+export function isNumber(variable: string) {
+  return (
+    variable !== undefined &&
+    variable !== null &&
```

Comment:
```suggestion
    variable != null
```
이 조건으로 null undefined를 한번에 검사할 수 있어요